typings/index.d.ts

```diff
@@ -115,12 +115,21 @@ declare module 'klasa' {
 		// Klasa Command Events
 		public on(event: 'commandError', listener: (msg: CommandMessage, command: Command, params: any[], error: Error) => void): this;
 		public on(event: 'commandInhibited', listener: (msg: CommandMessage, command: Command, response: string|Error) => void): this;
+		public on(event: 'commandRun', listener: (msg: CommandMessage, command: Command, params: any[], response: any) => void): this;
+		public on(event: 'commandUnknown', listener: (msg: DiscordMessage, command: string) => void): this;
 
 		// Klasa Console Custom Events
 		public on(event: 'log', listener: (data: any, type: string) => void): this;
 		public on(event: 'wtf', listener: (failure: Error) => void): this;
 		public on(event: 'verbose', listener: (data: any) => void): this;
 
+		// Klasa Piece Events
+		public on(event: 'pieceLoaded', listener: (piece: Piece) => void): this;
```

Comment:
I'd suggest against the use of Piece since it's an abstract class and shall not be used directly, however, they get instances of each kind of piece.

Reply:
Welp what to use then considering Piece is all pieces aka Commands, Events, Languages, Monitors etc

Reply:
I guess I could do `Commnad|Monitor|Language|Event` etc

Reply:
Oh, that's right... keep that.